WordPress/gutenberg
# [RNMobile] - Starter Page Templates - Picker and Preview design updates

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2027

## Description

This PR addresses some [design feedback](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2012) for the Starter Page Templates Picker and Preview Modal:

- [iOS] Dark Mode: border-bottom on the navigation bar is a bit too strong. We can probably dim it down a bit?
- [iOS] Dark Mode: I wonder if we should apply a dark gray bg on the Navigation Bar to match the native one.
- Can we apply a 300ms fadeOut to the inline picker buttons when they disappear?

It also solves this [transparency issue](https://github.com/WordPress/gutenberg/pull/20718#issuecomment-596708194).

## Screenshots 
FadeOut Animation iOS | FadeOut Animation iOS Dark Mode
-|-
| <img src="https://user-images.githubusercontent.com/4885740/76869651-e8cf2b00-6868-11ea-86f5-5417afd1bd63.gif" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/76869813-203dd780-6869-11ea-9c73-5949b4139886.gif" width=250 />  |

FadeOut Animation Android | Preview Modal Dark Mode iOS
-|-
| <img src="https://user-images.githubusercontent.com/4885740/76870221-b83bc100-6869-11ea-9c9d-8bb712c94ac5.gif" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/76870607-35673600-686a-11ea-8e3d-0cae5e0fd963.png" width=250 /> |

Buttons transparency fix iOS | Buttons transparency fix Android
-|-
| <img src="https://user-images.githubusercontent.com/4885740/76870365-e91bf600-6869-11ea-92ca-868a8855e0c3.png" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/76870371-eb7e5000-6869-11ea-92e5-c9cf3f8b4ced.png" width=250 />  |


## How has this been tested?
### Test 1
- Open the app with metro running
- Start a new page
- **Expect** The templates picker to be visible
  - Select a template
  - **Expect** the preview modal to be shown with the template
     - On iOS switch to dark mode and expect to see the new styles from this PR
  - Tap on Apply
  - **Expect** the modal to be dismissed and the picker to fade out.

### Test 2 [iOS]
- Open the app with metro running
- Start a new page
- **Expect** The templates picker to be visible
- Change the orientation to landscape
- Focus on the paragraph block
- **Expect** the buttons to have a solid background instead of having transparency.

## Types of changes
Design improvements

## Checklist:
